config/crd/bases/toolchain.dev.openshift.com_masteruserrecords.yaml

```diff
@@ -24,7 +24,7 @@ spec:
     - jsonPath: .status.conditions[?(@.type=="Ready")].reason
       name: Reason
       type: string
-    - jsonPath: .spec.userAccounts[].targetCluster
+    - jsonPath: .status.userAccounts[].cluster.name
```

Comment:
:thinking: this change is already in master
https://github.com/codeready-toolchain/host-operator/blob/940380ffbeecb3b6c89d29a894751c5749469f9d/config/crd/bases/toolchain.dev.openshift.com_masteruserrecords.yaml#L27
committed in https://github.com/codeready-toolchain/host-operator/commit/05a18d7aac4152738f42339b55c7d77feee350d7